bitnami/contour/values.yaml

```
@@ -0,0 +1,234 @@
+#   imagePullSecrets:
+#     - myRegistryKeySecretName
+
+## To configure Contour, you must specify ONE of the following two
+## options.
+#
+## existingConfigMap specifies the name of an externally-defined
+## ConfigMap to use as the configuration. Helm will not manage the
+## contents of this ConfigMap, it is your responsibility to create it.
+#
+# existingConfigMap: contour
+#
+## configInline specifies Contour's configuration directly, in yaml
+## format. When configInline is used, Helm manages Contour's
+## configuration ConfigMap as part of the release, and
+## existingConfigMap is ignored.
+## Refer to https://projectcontour.io/docs/v1.2.1/configuration/ for
+## available options.
+#
+configInline:
+  # should contour expect to be running inside a k8s cluster
+  # incluster: true
+  #
+  # path to kubeconfig (if not running inside a k8s cluster)
+  # kubeconfig: /path/to/.kube/config
+  #
+  # Client request timeout to be passed to Envoy
+  # as the connection manager request_timeout.
+  # Defaults to 0, which Envoy interprets as disabled.
+  # Note that this is the timeout for the whole request,
+  # not an idle timeout.
+  # request-timeout: 0s
+  # disable ingressroute permitInsecure field
+  disablePermitInsecure: false
+  tls:
+  #   minimum TLS version that Contour will negotiate
+  #   minimum-protocol-version: "1.1"
+  # The following config shows the defaults for the leader election.
+  ## This needs to be edited by when you deploy to a namespace other than projectcontour
+  # leaderelection:
+  #   configmap-name: leader-elect
+  #   configmap-namespace: projectcontour
+  ### Logging options
+  # Default setting
+  accesslog-format: envoy
+  # To enable JSON logging in Envoy
+  # accesslog-format: json
+  # The default fields that will be logged are specified below.
+  # To customise this list, just add or remove entries.
+  # The canonical list is available at
+  # https://godoc.org/github.com/projectcontour/contour/internal/envoy#JSONFields
+  # json-fields:
+  #   - "@timestamp"
+  #   - "authority"
+  #   - "bytes_received"
+  #   - "bytes_sent"
+  #   - "downstream_local_address"
+  #   - "downstream_remote_address"
+  #   - "duration"
+  #   - "method"
+  #   - "path"
+  #   - "protocol"
+  #   - "request_id"
+  #   - "requested_server_name"
+  #   - "response_code"
+  #   - "response_flags"
+  #   - "uber_trace_id"
+  #   - "upstream_cluster"
+  #   - "upstream_host"
+  #   - "upstream_local_address"
+  #   - "upstream_service_time"
+  #   - "user_agent"
+  #   - "x_forwarded_for"
+
+## String to partially override contour.fullname include (will maintain the release name)
+##
+# nameOverride:
+
+## String to fully override contour.fullname template
+##
+# fullnameOverride:
+
+## Number of contour Pod replicas
+##
+replicaCount: 2
+
+rbac:
+  # create specifies whether to install and use RBAC rules.
+  create: true
+
+contour:
+  image:
+    registry: docker.io
+    repository: projectcontour/contour
+    tag: v1.2.1
```

Once finished the container on our side, it should be something like:
```suggestion
    registry: docker.io
    repository: bitnami/contour
    tag: 1.2.1-debian-10-r0
```